charms/kserve-controller/src/charm.py

```diff
@@ -60,6 +70,19 @@
     "src/templates/webhook_manifests.yaml.j2",
 ]
 
+# Values for MinIO manifests https://kserve.github.io/website/0.11/modelserving/storage/s3/s3/
+S3_USEANONCREDENTIALS = "false"
+S3_REGION = "us-east-1"
+S3_USEHTTPS = "0"
```

Comment:
Are we ok with hardcoding these?

Reply:
Yes for now. Opened [issue](https://github.com/canonical/kserve-operators/issues/186) to explain